webcompat/issues.py

```diff
@@ -23,12 +23,17 @@
 PRIVATE_REPO_URI = app.config['PRIVATE_REPO_URI']
 
 
-def unmoderated_issue():
+def moderation_template():
```

Comment:
Do we want this to take an argument, and it will return the "in moderation" vs "you're a bad person" body content?

(For public issue, it will always return "in moderation", but then we could re-use the method if something fails moderation later on)

Reply:
I will modify it probably when doing #3141 (aka #3140)

Reply:
yeah, sounds good to me